src/services/notifications/notifications.test.ts

```diff
@@ -0,0 +1,190 @@
+    });
+
+    after(async () => {
+      await service.remove(notificationId);
+      await app.service('users').remove(userId);
+    });
+
+    it('should succeed - create notification', async () => {
+      const notificationResponse = await service.create(notificationInfo);
+      const { description, id } = notificationResponse;
+      notificationId = id;
+      expect(id).to.exist;
+      expect(description).to.equal(notificationInfo.description);
+      expect(notificationResponse).to.have.keys(['id', 'isRead', 'type', 'description', 'createdAt', 'updatedAt', 'userId']);
+    });
+
+    it('should fail - not existed userId', async () => {
+      const notificationInvalidUserId = {
+        description: 'test notification',
+        userId: '783deee0-3732-11ec-9301-13adbda06b66',
+      };
+      try {
+        await service.create(notificationInvalidUserId);
+        expect.fail('call should have failed');
+      } catch (err: any) {
+        const { code } = err;
+        expect(code).to.equal(400);
+      }
+    });
+
+    it('should fail - invalid data', async () => {
+      const notificationInvalidData = {
+        description: 12,
+        isRead: 'yes',
+        type: 'READ_WRITE',
+      };
+      try {
+        await service.create(notificationInvalidData);
+        expect.fail('call should have failed');
+      } catch (err: any) {
+        const { code } = err;
+        expect(code).to.equal(400);
+      }
+    });
+  });
+
+  describe('get notification by id', () => {
+    before(async () => {
+      const response = await app.service('users').create(userInfo);
+      userId = response.id;
+      notificationInfo.userId = response.id;
+      const { id } = await service.create(notificationInfo);
+      notificationId = id;
+    });
+
+    after(async () => {
+      await service.remove(notificationId);
+      await app.service('users').remove(notificationInfo.userId);
+    });
+
+    it('should succeed - get notification by id', async () => {
+      const notificationResponse = await service.get(notificationId);
+      const { description, type, id, userId } = notificationResponse;
+      expect(description).to.equal(notificationInfo.description);
+      expect(type).to.equal(notificationInfo.type);
+      expect(userId).to.equal(userId);
+      expect(id).to.equal(notificationId);
+    });
+
+    it('should fail - not existed notificationId', async () => {
+      try {
+        await service.get(notExistedNotificationId);
+        expect.fail('call should have failed');
+      } catch (err: any) {
+        const { code } = err;
+        expect(code).to.equal(404);
+      }
+    });
+
+    it('should fail - invalid id', async () => {
+      try {
+        await service.get('7777');
+        expect.fail('call should have failed');
+      } catch (err: any) {
+        const { code } = err;
+        expect(code).to.equal(400);
+      }
+    });
+  });
+
+  describe('find notification', () => {
```

Comment:
add test, where you use pagination, check sorting

Reply:
tests added 9978e7fc0bb98929f17ca5c20384f8b3d56f0792